app/assets/stylesheets/default.scss

```diff
@@ -45,4 +45,68 @@ br {
 .btn-primary.change-red {
     background-color: red !important;
     border-color: red;
-}
\ No newline at end of file
+}
+
+// .btn-primary {
+//     color:#fff;
+//     background-color:#007bff;
+//     border-color:#007bff;
+// }
+
+// .btn-primary:hover {
+//     color:#fff;
+//     background-color:#0069d9;
+//     border-color:#0062cc;
+// }
+
+// .btn-primary.focus,.btn-primary:focus {
+//     box-shadow:0 0 0 .2rem rgba(0,123,255,.5);
+// }
+
+// .btn-primary.disabled,.btn-primary:disabled {
+//     color:#fff;
+//     background-color:#007bff;
+//     border-color:#007bff;
+// }
+
+// .btn-primary:not(:disabled):not(.disabled).active,.btn-primary:not(:disabled):not(.disabled):active,.show>.btn-primary.dropdown-toggle {
+//     color:#fff;
+//     background-color:#0062cc;
+//     border-color:#005cbf;
+// }
+
+// .btn-primary:not(:disabled):not(.disabled).active:focus,.btn-primary:not(:disabled):not(.disabled):active:focus,.show>.btn-primary.dropdown-toggle:focus {
+//     box-shadow:0 0 0 .2rem rgba(0,123,255,.5);
+// }
+
+// .btn-secondary {
+//     color:#fff;
+//     background-color:#6c757d;
+//     border-color:#6c757d
+// }
+
+// .btn-secondary:hover {
+//     color:#fff;
+//     background-color:#5a6268;
+//     border-color:#545b62;
+// }
+
+// .btn-secondary.focus,.btn-secondary:focus {
+//     box-shadow:0 0 0 .2rem rgba(108,117,125,.5);
+// }
+
+// .btn-secondary.disabled,.btn-secondary:disabled {
+//     color:#fff;
+//     background-color:#6c757d;
+//     border-color:#6c757d;
+// }
+
+// .btn-secondary:not(:disabled):not(.disabled).active,.btn-secondary:not(:disabled):not(.disabled):active,.show>.btn-secondary.dropdown-toggle {
+//     color:#fff;
+//     background-color:#545b62;
+//     border-color:#4e555b;
+// }
+
+// .btn-secondary:not(:disabled):not(.disabled).active:focus,.btn-secondary:not(:disabled):not(.disabled):active:focus,.show>.btn-secondary.dropdown-toggle:focus {
+//     box-shadow:0 0 0 .2rem rgba(108,117,125,.5);
+// }
```

Comment:
Is all of this stuff commented out? Why is it included?

Reply:
I was including some bootstrap source code to fix buttons but didn't use it, but also don't want to lose it.

Reply:
Cool, makes sense